src/docs/sdk/features.mdx

```diff
@@ -231,6 +231,58 @@ If Performance Monitoring is both supported by the SDK and enabled in the client
 - span status must match HTTP response status code ([see Span status to HTTP status code mapping](/sdk/event-payloads/span/))
 - when network error occurs, span status must be set to `internal_error`
 
+### HTTP Client errors
+
+HTTP Client errors are automatically captured and sent to `https://sentry.io`.
+
+The HTTP Client integration should have 3 configuration options:
+
+* `captureFailedRequests` defaults to `false` due to PII reasons.
+* `failedRequestStatusCode` defaults to `500` to `599`, this configuration option accepts a `List` of `HttpStatusCodeRange` which is a range of HTTP status code -> `min` to `max` or a single `status_code`.
```

Comment:
```suggestion
* `failedRequestStatusCodes` defaults to `500` to `599`, this configuration option accepts a `List` of `HttpStatusCodeRange` which is a range of HTTP status code -> `min` to `max` or a single `status_code`.
```
Should this be code"s" since it's a list of ranges?